### PR TITLE
Update lsfr-related links, and more

### LINKED ISSUE #343: Rearrange aws docs
Per #61, we're going to consolidate all aws and lsfr docs to _all_ live in `aws`. 
We'll need to make the following changes:

- [ ] Replace the `aws/index.md` file with an overview of S3 and LSfR
- [ ] Move the current `aws/index.md` content into a new file about joining AWS
- [ ] Update relative links across docs

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #343 

This PR updates links following the deletion of `lsfr/index.md` (and a couple more I noticed along the way). Note that some relative links point to pages which are still in PRs, so I'm leaving this as a draft until everything linked has gone in. Then, I'll check again that there are no link warnings before marking this ready for review.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
#349 is in!

**Comment by sjspielman** [comment]
This is the only broken link right now 😎 

```
INFO    -  Doc file 'software-platforms/aws/index.md' contains an unrecognized
           relative link 'STUB_LINK LSfR snapshots', it was left as is.
```

File: docs/software-platforms/aws/joining-aws.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ You must accept this invitation to gain access to your AWS account and to downlo
 To accept the invitation, please take the following steps.
 
 1. Note the following information in the invitation email: **Your AWS access portal URL**.
-   You will need this value to [configure AWS CLI locally](../../technical-setup/environment-setup/configure-aws-cli.md) and to [log into the console to use Lightsail for Research](STUB_LINK).
+   You will need this value to [configure AWS CLI locally](../../technical-setup/environment-setup/configure-aws-cli.md) and to [log into the console to use Lightsail for Research](./creating-vcs.md#how-to-create-a-virtual-computer). TODO IS THIS RELATIVE LINK CORRECT??????
```
Comment (allyhawkins) [inline docs/software-platforms/aws/joining-aws.md:12]:
```suggestion
   You will need this value to [configure AWS CLI locally](../../technical-setup/environment-setup/configure-aws-cli.md) and to [log into the console to use Lightsail for Research](./creating-vcs.md#how-to-create-a-virtual-computer).
```
I think this should be correct?

**Review by allyhawkins — APPROVED** [review]
LGTM!

**Comment by sjspielman** [comment]
Hm it appears I have committed a workflow file to the wrong branch here...

**Pull request merged**